src/site/_data/paths/secure.json

```diff
@@ -48,8 +48,6 @@
         "samesite-cookie-recipes",
         "referrer-best-practices",
         "trust-tokens",
-        "conversion-measurement",
```

Comment:
Does this mean that the correspondinf articles should be removed too? e.g. https://github.com/GoogleChrome/web.dev/blob/main/src/site/content/en/blog/conversion-measurement/index.md

Reply:
Yes, these can be removed